.github/workflows/tests.yml

```
@@ -42,8 +42,8 @@ jobs:
       - name: Execute Test
         run: python ./.github/tests/encoding_test.py
 
-  check_whiteSpace_comma:
-    name: Check no whitespace before a comma
+  Whitespace_Comma:
+    name: Check For Whitespaces Before A Comma
```

```suggestion
    name: Check For Whitespaces Before Comma
```

Finde das unnötig lang